### PR TITLE
Lab 10 MNIST and High-level TF API(dropout, batchnorm, xavier)

### PR DESCRIPTION
**Pull request by jihobak**
I would like to show powerful of High-level API built in Tensorflow.
So, I wrap up Lab MNIST with TF API.

I hope this help people to get used to make compact code after studying fundamental tf code.

Thanks,

### CODE REVIEW

**Comment by hunkim** [comment]
@jihobak We look forward to this file. Can you push and/or send us another PR?
@kkweon How this is possible? github bug?

**Comment by jihobak** [comment]
@hunkim @kkweon 
I'm sorry, It was my fault. 
Could you check again plz?

Thanks a lot

File: lab-10-7-mnist_nn_higher_level_API.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+mnist = input_data.read_data_sets("MNIST_data/", one_hot=True)
+# Check out https://www.tensorflow.org/get_started/mnist/beginners for
+# more information about the mnist dataset
+
+# parameters
+learning_rate = 0.01  # we can use large learning rate using Batch Normalization
+training_epochs = 15
+batch_size = 100
+train_mode = tf.placeholder(tf.bool, name='train_mode')
```
Comment (kkweon) [inline lab-10-7-mnist_nn_higher_level_API.py]: IMO, all placeholders should be grouped together 
like
```python
X = tf.placeholder(tf.float32, [None, 784])
Y = tf.placeholder(tf.float32, [None, 10])
train_mode = tf.placeholder(tf.bool, name='train_mode')
```

**Comment by hunkim** [comment]
@jihobak Could you remove pytorch files in this PR? Usually, it's a good idea to make a branch for each pr.

File: lab-10-7-mnist_nn_higher_level_API.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+    for i in range(total_batch):
+        batch_xs, batch_ys = mnist.train.next_batch(batch_size)
+        feed_dict_train = {X: batch_xs, Y: batch_ys, train_mode: True}
+        feed_dict_cost = {X: batch_xs, Y: batch_ys, train_mode: False}
+        opt = sess.run(optimizer, feed_dict=feed_dict_train)
+        c = sess.run(cost, feed_dict=feed_dict_cost)
+        avg_cost += c / total_batch
+
+    print('Epoch:', '%04d' % (epoch + 1), 'cost =', '{:.9f}'.format(avg_cost))
```
Comment (kkweon) [inline lab-10-7-mnist_nn_higher_level_API.py]: IMO, this print format is not pythonic.

It should be written in f-string or format
```python
print(f"[Epoch: {epoch + 1}] cost = {avg_cost}")
print("[Epoch: {}] cost = {}".format(epoch + 1, avg_cost))

#right-alignment(>) and digit format
print(f"[Epoch: {epoch + 1:>4}] cost = {avg_cost:>.9}")
print("[Epoch: {:>4}] cost = {:>.9}".format(epoch + 1, avg_cost))
```

**Comment by jihobak** [comment]
@hunkim Okay, Thanks :)

File: lab-10-7-mnist_nn_higher_level_API.py (hunk 2 above)
Comment (jihobak) [inline lab-10-7-mnist_nn_higher_level_API.py]: @kkweon @hunkim 
I agree with you but I just keep it as other code use this format.
Should I change it and commit again?

File: lab-10-7-mnist_nn_higher_level_API.py (hunk 1 above)
Comment (jihobak) [inline lab-10-7-mnist_nn_higher_level_API.py]: @kkweon great ! :) thankss

File: lab-10-7-mnist_nn_higher_level_API.py (hunk 2 above)
Comment (kkweon) [inline lab-10-7-mnist_nn_higher_level_API.py]: Not sure, @hunkim should decide.

Anyway, other codes are just waiting for PRs too.
So, I guess it should be a separate PR since you have already committed.

Comment (jihobak) [inline lab-10-7-mnist_nn_higher_level_API.py]: @kkweon Thanks for your advice :)

Comment (hunkim) [inline lab-10-7-mnist_nn_higher_level_API.py]: Let's do with *pythonic.*

Comment (hunkim) [inline lab-10-7-mnist_nn_higher_level_API.py]: Let's merge for now and @jihobak please send another PR for pythonic.

Comment (jihobak) [inline lab-10-7-mnist_nn_higher_level_API.py]: @hunkim Thanks for review, I'm going to make PR today.
